lib/Parser/ParseFlags.h

```diff
@@ -51,6 +51,9 @@ enum
                                      //  let/const in global scope instead of eval scope so that they can be preserved across console inputs
     fscrNoAsmJs = 1 << 25, // Disable generation of asm.js code
     fscrIsModuleCode = 1 << 26, // Current code should be parsed as a module body
-    fscrAll = (1 << 27) - 1
+
+    fscrIndirect = 1 << 27,
```

Comment:
Is it to indicate indirect eval? If so, then we already have flags that do that. fscrEval indicates direct eval, i.e., interaction with the caller's context. fsrcEvalCode && !fscrEval indicates indirect eval interacting with the global context. (Feel free to rename them for clarity if you want.)